CMakeLists.txt

```diff
@@ -283,28 +283,26 @@ add_custom_command(OUTPUT ${stamp_file}
                    COMMAND ${CMAKE_COMMAND} -E touch ${stamp_file}
                    COMMENT "Copying directories such as etc, icons, fonts, js, ui5, etc. to build area")
 
+#---- List of directories which completely copied to target directory by each build
+set(artifact_dirs "etc;test;icons;fonts;macros;tutorials")
```

Comment:
You can probably also move the `README` directory from above (line 282) into this list.

Reply:
Done